code/game/objects/items/weapons/storage/secure_vr.dm

```diff
@@ -0,0 +1,4 @@
+/obj/item/weapon/storage/secure/briefcase/toxinsfootball
+	name = "toxins football"
+	desc = "A large briefcase with a digital locking system and sleek, sciency design. This one has a hazard symbol that warns of explosive contents. It feels extremely heavy."
+	force = 14 // DON'T FUCK WITH THE SCIENCE TEAM!
```

Comment:
tbh if it is going to have more force, it probably should have slowdown as well.